### PR TITLE
DSKaKhel: Queue implementation using Java

### PR DESCRIPTION
**Pull request by ghost**
# HacktoberFest 22

DSKaKhel: Queue

## Programming Language for Data Structure Implementation

Java

## Code file name with extension

queue.java

## What was the toughest question you got till now in the interviews ?

Why do we hire you?

## YOU ARE

a student 

## GITHUB USERNAME

mearchit

### We are looking for a cool name for this repository for next hacktober fest please suggest below (optional)

`_____________`

Thanks for your contributions !!

### CODE REVIEW

**Comment by GauravWalia19** [comment]
Can you please fill the PR template, it seems some details are missing

**Comment by ghost** [comment]
PR template updated

File: queues/QUEUE.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        {
+            for(int i=front;i<=rear;i++){
+                System.out.print(arr[i]+"\t");
+            }
+        }
+        System.out.println();
+    }
+}
+
+public class QUEUE {
```
Comment (GauravWalia19) [inline queues/QUEUE.java]: Please rename it as Queue and rename file as Queue.java

File: queues/QUEUE.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import java.util.Scanner;
+class q_back
```
Comment (GauravWalia19) [inline queues/QUEUE.java]: Please rename it as ArrayQueue and rename the other ocurrances as well

**Review by GauravWalia19 — CHANGES_REQUESTED** [review]
## Changes requested

1. Do the commented changes
2. Move the file to location: `queues/array_queue/Queue.java`
3. Please update the README as well

**Pull request closed without merging**